torch/csrc/jit/codegen/cuda/kernel.cpp

```diff
@@ -358,6 +361,73 @@ std::vector<Expr*>& KernelInternalProxy::topLevelExprs() {
   return kernel_->top_level_exprs_;
 }
 
+void KernelPerformanceProfile::registerExpr(const Expr* expr) {
+  if (expr_entry_map_.find(expr) != expr_entry_map_.end()) {
+    return;
+  }
+
+  auto slot = getNewIndex();
+  expr_entry_map_.emplace(expr, slot);
+}
+
+int KernelPerformanceProfile::getNewIndex() {
+  auto slot = num_profile_entries_;
+  ++num_profile_entries_;
+  return slot;
```

Comment:
Nit: isn't this just `return num_profile_entries_++;`

Reply:
Yes. I think I was doing something more but ended up just with this. Cleaned it up.